main.go

```diff
@@ -409,7 +387,7 @@ func runWatcher(fileChangeCommand string, dirs []string, killServer bool) error
 	errorMessage := ""
 	var err error
 
-	Debug.log("Run watcher ENTRY: " + fileChangeCommand)
+	ControllerDebug.log("Starting watcher for ON_CHANGE command: " + fileChangeCommand)
```

Comment:
At this point you are just starting the watcher correct? not actually executing the `fileChangeCommand`? If so, I think you should just log "Starting watcher" here, it is confusing to log the command that is not run yet.